### ISSUE TITLE
DataProvider Parallel tests run sequentially.

### ISSUE THREAD

**Issue by wildcat111** [qmetry/qaf#110]
I am trying to run a Single test with multiple data sets, this test will be run over a thousand times using the data provider. I want all these executions to run in parallel. I am using the parallel=true on the data provider and in a normal selenium setup this is working fine, but when running the same test in a Qmetry setup the tests run sequentially.

**Comment by wildcat111** [comment]
Going through some other issues, I found a property that must be set for this to work in Qmetry.  

global.datadriven.parallel=1

This property should be set to 1 for DataProvider Parallel to work. This is not mentioned in any documentation anywhere Qmetry, I just happened to come across another issue in this Repo that mentioned it.

**Comment by cjayswal** [comment]
You Got it. You are wellcome to edit documentation and do pull request for benefit of yourself and others.

**Comment by wildcat111** [comment]
@cjayswal, I'd be more than happy to add this, but do you know if there is a complete list of all the properties that can be set for Qmetry? I am not seeing this property mentioned in the documentation here: 

https://qmetry.github.io/qaf/latest/docs.html
